emu_mps/hamiltonian.py

```diff
@@ -361,6 +362,9 @@ def make_H(
     nqubits = interaction_matrix.size(dim=1)
     middle = nqubits // 2
 
+    assert is_symmetric_zero_diag_matrix(
+        interaction_matrix
```

Comment:
this is a torch tensor, not a `list[list[float]]`, from the type annotation of `make_H`
- why doesn't mypy complain?
- this check deserves existing but maybe put it in `emu_base/base_classes/config.py:53` as you can see there is already some validation there
- you can check in [Henrique's MR](https://github.com/pasqal-io/Pulser/pull/764) if/how he did it there

Reply:
It seems that it will be easier to add several checks on his side rather than emu-mps

Reply:
let's put the checks in emu_base config for now. Those files will be deleted naturally when we move to the pulser files.

Reply:
moved